xlators/features/locks/src/posix.c

```diff
@@ -93,9 +93,14 @@ fetch_pathinfo(xlator_t *, inode_t *, int32_t *, char **);
             dict_unref(__unref);                                               \
     } while (0)
 
+#define IS_BIT_SET(byte, bit_number) (byte & (1 << bit_number))
+
+#define SET_BIT(byte, bit_number) (byte = (byte | (1 << bit_number)))
```

Comment:
Modifying arguments inside a macro is a bad practice because it's a hidden modification and because `byte` could be anything, even thing that cannot appear in the left side of an assignment.